### PR TITLE
Add support for FreeBSD 11.2

### PR DESCRIPTION
**Pull request by ophum**
- Add support for FreeBSD 11.2
- Fix install_authorized_keys.sh

## Fix details

`sed` command behavior is different between 11.2 and 12+.

in install_authorized_keys.sh

https://github.com/cross-platform-actions/freebsd-builder/blob/v0.8.0/resources/provision.sh#L52
```
... | sed 's/ /\n/g' | ...
```

11.2 behavior is not replace space to newline.
```
% freebsd-version
11.2-RELEASE
% echo "This is a test" | sed 's/ /\n/g'
Thisnisnantest
```

this solution is using `tr` command instead of `sed` command.

```
% freebsd-version
11.2-RELEASE
% echo "This is a test" | tr ' ' '\n'
This
is
a
test
```

## build

forked repo's actions is all succeeded

https://github.com/ophum/freebsd-builder/actions/runs/9860845096

## test

test ci is run `uname -a` at all version.
https://github.com/ophum/freebsd-actions-test/blob/main/.github/workflows/ci.yaml

result: 
https://github.com/ophum/freebsd-actions-test/actions/runs/9862512116

### CODE REVIEW

File: resources/provision.sh [hunk 1]
```diff
@@ -111,7 +111,7 @@ bootstrap_pkg() {
 }
 
 install_local_package() {
-  ASSUME_ALWAYS_YES=yes pkg add "/mnt/packages/FreeBSD:12:amd64/All/$1"-[0123456789]*
+  ASSUME_ALWAYS_YES=yes pkg add "/mnt/packages/FreeBSD:$ABI_VERSION:$PKG_SITE_ARCHITECTURE/All/$1"-[0123456789]*
```
Comment (ophum) [inline resources/provision.sh:114]: this function will also be used 11.2, so use variables.

**Comment by jacob-carlborg** [comment]
Hmm, isn't FreeBSD 11 obsolete? I had to jump through hoops to not break the action and remove support for FreeBSD 12 when its package repository was removed. As far as I understand you cannot install any packages, unless you can find a mirror still keeping FreeBSD 11 packages. I guess you can use a custom image and install packages during provisioning.

**Comment by ophum** [comment]
yes, I think FreeBSD 11 is obsolete. 

I need FreeBSD 11.2. but I can build custom image as you say.  so, I think it's okay to close this pr.
I hope this pr remains as knowledges for others who need it. 👍

**Comment by jacob-carlborg-apoex** [comment]
> I need FreeBSD 11.2. but I can build custom image as you say.

Be aware that there's no guarantee that a custom version will work in the future, or at all. Custom images are intended for doing work ahead of time of existing supported operating systems and versions.

**Comment by jacob-carlborg-apoex** [comment]
Be aware that there's no guarantee that a custom version will work in the future, or at all. Custom images are intended for doing work ahead of time of existing supported operating systems and versions.

**Comment by jacob-carlborg** [comment]
Be aware that there's no guarantee that a custom version will work in the future, or at all. Custom images are intended for doing work ahead of time of existing supported operating systems and versions.

**Pull request closed without merging**